massgov/mayflower-patternlab
# DP-4572 - Re-work Dateline

## Description
I reworked the Dateline for the Drupal side. FYI -- I kept the `<p>` tag for current PL structure, but we will not be using this pattern on the Drupal end. (On the Drupal end, we're separating everything out and adding the Dateline, and reassembling programmatically.) 

## Related Issue / Ticket

- https://jira.state.ma.us/browse/DP-4572
- [Github issue]()

## Steps to Test


1. Read Ticket
2. go to *?p=pages-press-release*
3. confirm span and bolded dateline.

## Screenshots
![screen shot 2017-09-18 at 11 29 29 am](https://user-images.githubusercontent.com/231467/30550221-cc4d01a0-9c64-11e7-811e-eebe4a33ecc0.png)



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
